building/upload-debs/conf/updates

```diff
@@ -0,0 +1,4 @@
+Name: homeworld
+Components: main
+Method: http://web.mit.edu/hyades/debian/
```

Comment:
We probably want to change the upstream branch from `/hyades/debian/` to `/hyades/apt/root/master/` or something. Currently, the modified rebuild.sh script can't actually publish to the upstream branch...

Reply:
Configuring upstream is pretty much just changing this updates file; I don't think it's necessary to integrate it into our build process. In fact, if I understand reprepro correctly, you can define multiple upstream repositories here under different names, then it'd just be a matter of changing the name in the "Update:" field in conf/distributions to decide which one to use.

Could you move /hyades/debian/ to /hyades/apt/root/master/? (It's currently empty.) I don't have the bits for that.

Reply:
I've moved the folder, and left a symlink from /hyades/debian.

Your thoughts on configurable upstreams seem reasonable. Good enough for me.